clients/client-python/gravitino/utils/http_client.py

```diff
@@ -65,8 +65,6 @@ def headers(self):
     def json(self):
         if self.body:
             return _json.loads(self.body.decode("utf-8"))
-        else:
-            return None
```

Comment:
Why we need deleted these codes?

Reply:
Pylint thinks it is redundant with no-else-return rule. The default return of Python function is `None`, there is no need to return explicitly.

Reply:
OK, Thank you for your reply.